src/libdrakvuf/libdrakvuf.h

```diff
@@ -202,6 +202,9 @@ typedef struct drakvuf_trap_info
     };
 } drakvuf_trap_info_t;
 
+#define UNLIMITED_TTL -1
```

Comment:
If you were to use `0` as a placeholder for `unlimited` then you could drop having to set the `cpuid.ttl` to `UNLIMITED_TTL` explicitely. It would just be the default if the field is not specifically initialized